src/node/kernel_notifications.cpp

```diff
@@ -57,9 +58,14 @@ static void DoWarning(const bilingual_str& warning)
 
 namespace node {
 
-void KernelNotifications::blockTip(SynchronizationState state, CBlockIndex& index)
+kernel::InterruptResult KernelNotifications::blockTip(SynchronizationState state, CBlockIndex& index)
```

Comment:
This requires to `#include "kernel/notifications_interface.h"`, no?

Reply:
> This requires to `#include "kernel/notifications_interface.h"`, no?

It is included from the header, and is not a new dependency since this file is subclassing that one, but yes according to IWYU's reasoning it should be included directly here as well. I'll add the include if I need to update this PR for another reason.